src/Compilers/CSharp/Portable/Binder/ExpressionVariableFinder.cs

```diff
@@ -441,6 +441,36 @@ internal class ExpressionVariableFinder : ExpressionVariableFinder<LocalSymbol>
             s_poolInstance.Free(finder);
         }
 
+        /// <summary>
+        /// Count the number of expression variables declared among the nodes.
+        /// </summary>
+        internal static int CountExpressionVariables(
+            Binder scopeBinder,
```

Comment:
Should I make this parameter a compilation and create the dummy binder locally rather than in the caller?